### PR TITLE
fix: children elements in nested lists drag parent elements.

### PR DESCRIPTION
**Pull request by DeadHeadRussell**
Here is a pull request for issue #112 

This commit has both a fix, and also a new story for nested lists as a test case.

### CODE REVIEW

File: src/.stories/index.js [hunk 1]
```diff
@@ -27,14 +27,26 @@ const Item = SortableElement((props) => {
...
 });
 
+const Category = SortableElement((props) => {
+  return (
+    <div className={style.category}>
+      <div className={style.categoryHeader}>
+        <Handle/>
+        <span>Category {props.value}</span>
+      </div>
+			<ListWrapper component={SortableList} className={style.categoryList} items={getItems(5, 59)} shouldUseDragHandle={true} helperClass={style.stylizedHelper} />
```
Comment (tomasztomys) [inline src/.stories/index.js]: wrong indent

File: src/SortableContainer/index.js [hunk 2]
```diff
@@ -143,6 +143,10 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
 			}
 		};
 
+    nodeIsChild = node => {
```
Comment (tomasztomys) [inline src/SortableContainer/index.js]: wrong indent

File: src/.stories/index.js (hunk 1 above)
Comment (DeadHeadRussell) [inline src/.stories/index.js]: fixed.

File: src/SortableContainer/index.js (hunk 2 above)
Comment (DeadHeadRussell) [inline src/SortableContainer/index.js]: fixed.

**Comment by clauderic** [comment]
Hey @DeadHeadRussell, thank you so much for taking the time to submit this, and for even going the extra mile and writing a storybook demo for it ❤️ Should be released shortly, stay tuned!

**Comment by DeadHeadRussell** [comment]
No problem. Thanks for merging it so quickly!

**Pull request merged**